packages/quick_actions/android/src/main/java/io/flutter/plugins/quickactions/MethodCallHandlerImpl.java

```diff
@@ -0,0 +1,133 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.quickactions;
+
+import android.annotation.TargetApi;
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.ShortcutInfo;
+import android.content.pm.ShortcutManager;
+import android.content.res.Resources;
+import android.graphics.drawable.Icon;
+import android.os.Build;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+class MethodCallHandlerImpl implements MethodChannel.MethodCallHandler {
+
+  private static final String CHANNEL_ID = "plugins.flutter.io/quick_actions";
+  private static final String EXTRA_ACTION = "some unique action key";
```

Comment:
Should "some unique action key" be written here?

Reply:
I don't have enough context to give input here. It seems this is introduced in https://github.com/flutter/plugins/commit/63609c94779331977d9492fc89cd6fb56cc68256 
and @collinjackson or @BugsBunnyBR might be able to give some inputs on this.
Anyways, this is not related to this migration. I suggest we create new issue and fix it in a later PR if needs to be fixed.

Reply:
That is just a unique string, the value itself does not matter.

Reply:
@BugsBunnyBR We can make the value of the String unambiguous so it is more readable.